### ISSUE TITLE
Unable to create a batch change: `Preparing container images  panic: strings: negative Repeat count`

### ISSUE THREAD

**Issue by szymi-** [sourcegraph/src-cli#629]
I am getting the following error when trying to create a batch change:

  ```$ src batch preview -f batch_change.yaml
  ✅ Parsing batch spec
  ✅ Resolving namespace
  ⠋  Preparing container images  panic: strings: negative Repeat count
  
  goroutine 1 [running]:
  strings.Repeat({0x16f8096, 0xc0000c2010}, 0x16f7b0a)
          /opt/hostedtoolcache/go/1.17.1/x64/src/strings/strings.go:529 +0x4d8
  github.com/sourcegraph/sourcegraph/lib/output.(*progressTTY).writeBar(0xc00001a000, 0xc000010060)
          /home/runner/go/pkg/mod/github.com/sourcegraph/sourcegraph/lib@v0.0.0-20211011135305-1971ff7fa57e/output/progress_tty.go:254 +0x3bb
  github.com/sourcegraph/sourcegraph/lib/output.(*progressTTY).draw(0xc00001a000)
          /home/runner/go/pkg/mod/github.com/sourcegraph/sourcegraph/lib@v0.0.0-20211011135305-1971ff7fa57e/output/progress_tty.go:202 +0x3f
  github.com/sourcegraph/sourcegraph/lib/output.(*progressTTY).drawInSitu(0xc00001a000)
          /home/runner/go/pkg/mod/github.com/sourcegraph/sourcegraph/lib@v0.0.0-20211011135305-1971ff7fa57e/output/progress_tty.go:208 +0x33
  github.com/sourcegraph/sourcegraph/lib/output.(*progressTTY).SetValue(0xc00001a000, 0x0, 0xfff8000000000000)
          /home/runner/go/pkg/mod/github.com/sourcegraph/sourcegraph/lib@v0.0.0-20211011135305-1971ff7fa57e/output/progress_tty.go:84 +0x99
  github.com/sourcegraph/src-cli/internal/batches/ui.(*TUI).PreparingContainerImagesProgress(0xc00009ccd0, 0xc000113830, 0xc000113888)
          /home/runner/work/src-cli/src-cli/internal/batches/ui/tui.go:75 +0x38
  github.com/sourcegraph/src-cli/internal/batches/service.(*Service).EnsureDockerImages(0xc00038cab0, {0x17ee8d8, 0xc000304fc0}, 0xc000540000, 0xc000113b30)
          /home/runner/work/src-cli/src-cli/internal/batches/service/service.go:157 +0x54
  main.executeBatchSpec({0x17ee8d8, 0xc000304fc0}, {0xc000384120, 0x0, {0x1803348, 0xc00038cab0}, {0x17f3450, 0xc0002ce1e0}})
          /home/runner/work/src-cli/src-cli/cmd/src/batch_common.go:252 +0x575
  main.init.5.func1({0xc00009c080, 0xc00009c080, 0x7})
          /home/runner/work/src-cli/src-cli/cmd/src/batch_preview.go:52 +0x3b9
  main.commander.run({0xc0002d1780, 0x6, 0xc000302a20}, 0xc0002b8360, {0x16fd6ba, 0x16}, {0x172cf1d, 0x246}, {0xc00009c070, 0x3, ...})
          /home/runner/work/src-cli/src-cli/cmd/src/cmd.go:97 +0x3e5
  main.init.1.func1({0xc00009c070, 0xc00009c070, 0x5})
          /home/runner/work/src-cli/src-cli/cmd/src/batch.go:33 +0x65
  main.commander.run({0xc000316000, 0x11, 0x60}, 0xc0000c4120, {0x16f8030, 0x1c084f8}, {0x172f69d, 0x463}, {0xc00009c060, 0x4, ...})
          /home/runner/work/src-cli/src-cli/cmd/src/cmd.go:97 +0x3e5
  main.main()
          /home/runner/work/src-cli/src-cli/cmd/src/main.go:68 +0xb7
```

I am on macOS Big Sur. Error persists regardless of which method I use to install `src`: brew or curl.

**Comment by mrnugget** [comment]
Hey @szymi-! Thanks for reporting. Is there any chance you can share the `steps` of your spec file? I'm wondering if there's something curious with the `container:` fields.

**Comment by szymi-** [comment]
The file I'm editing is pretty simple, I was following this document: https://docs.sourcegraph.com/batch_changes/how-tos/tracking_existing_changesets

Contents of the file is to the to the following effect (with actual organization and repo names in relevant places, and with multiple changes listed):

    name: track-migration
    description: Track all changesets related tomigration

    importChangesets:
    - repository: github.com/<private_organization>/<private_repo_name>
      externalIDs: [5]

I have two environment variables for `src` set: `SRC_ENDPOINT` and `SRC_ACCESS_TOKEN` pointing to our internal Sourcegraph instance. The configuration from the variables works (I tested it with a simple repo content search). I validated the file using a .yaml validation tool.
